Dockerfile

```diff
@@ -126,6 +126,9 @@ COPY --from=generate-build /pkg/machinery/config/types/v1alpha1/*_doc.go /pkg/ma
 FROM build AS base
 COPY ./go.mod ./go.sum ./
 COPY ./pkg/machinery/go.mod ./pkg/machinery/go.sum ./pkg/machinery/
+WORKDIR /src/pkg/machinery
+RUN go mod download
```

Comment:
From https://golang.org/doc/go1.16:

> When the main module's go.mod file declares go 1.16 or higher, the all package pattern now matches only those packages that are transitively imported by a package or test found in the main module. (Packages imported by tests of packages imported by the main module are no longer included.) This is the same set of packages retained by go mod vendor since Go 1.11.

Without explicit `go mod download`, `go mod tidy` below downloads machinery's test dependencies, and `grep` fails:
```
#165 0.135 go: downloading github.com/google/go-cmp v0.5.0
#165 0.352 go: downloading golang.org/x/sync v0.0.0-20201020160332-67f06af15bc9
#165 0.352 go: downloading github.com/onsi/gomega v1.10.1
```